pkgs/servers/baserow/default.nix

```diff
@@ -0,0 +1,150 @@
+{ lib
+, buildPythonPackage
+, fetchFromGitLab
+, regex
+, service-identity
+, itsdangerous
+, requests
+, redis
+, channels
+, psycopg2
+, gunicorn
+, django-cors-headers
+, django-celery-email
+, advocate
+, django-storages
+, pillow
+, faker
+, uvicorn
+, twisted
+, django
+, drf-jwt
+, cryptography
+, tqdm
+, celery-redbeat
+, drf-spectacular
+, websockets
+, asgiref
+, antlr4-python3-runtime
+, psutil
+, dj-database-url
+, django-health-check
+, celery
+, unicodecsv
+, django-celery-beat
+, django-redis
+, zipp
+, boto3
+, pycodestyle
+, flake8
+}:
+
+let
+
+  overrideVersions = [
+    "regex"
+    "service-identity"
+    "itsdangerous"
+    "requests"
+    "redis"
+    "channels"
+    "psycopg2"
+    "gunicorn"
+    "django-cors-headers"
+    "django-celery-email"
+    "advocate"
+    "django-storages"
+    "Pillow"
+    "Faker"
+    "uvicorn"
+    "Twisted"
+    "Django"
+    "drf-jwt"
+    "cryptography"
+    "tqdm"
+    "celery-redbeat"
+    "drf-spectacular"
+    "websockets"
+    "asgiref"
+    "antlr4-python3-runtime"
+    "psutil"
+    "dj-database-url"
+    "django-health-check"
+    "celery"
+    "unicodecsv"
+    "django-celery-beat"
+    "django-redis"
+    "zipp"
+    "boto3"
+  ];
+
+in
+buildPythonPackage rec {
+  pname = "baserow";
+  version = "1.10.1";
+
+  src = fetchFromGitLab {
+    owner = "bramw";
+    repo = pname;
+    rev = version;
+    sha256 = "sha256-5rWuaVjpj55bGfbkvN8i/lKUtPmdHC5ab4ViBHG67/M=";
+  };
+
+  sourceRoot = "source/backend";
+
+  postPatch = ''
+    # Ignore pinned package versions from upstream for modules specificed in
+    # the list above
+    cp requirements/base.in requirements/base.txt
+  '' + lib.concatMapStringsSep "\n"
+    (e: ''
+      sed -i requirements/base.txt -e 's/^${e}.*/${e}/'
+    '')
+    overrideVersions;
```

Comment:
Just overwrite it for everything. We don't want add another chore and maintain that list.